cli/defenseclaw/bootstrap.py

```
@@ -258,25 +261,83 @@ def _fresh_migration_state():
     )
 
 
-def _record_fresh_migration_retry(cfg: Config) -> str:
+def _expected_fresh_migration_state():
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_expected_fresh_migration_state`

(ANN202)